examples/svelte/src/utils.js

```diff
@@ -0,0 +1,7 @@
+export function uuid() {
+    return "xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx".replace(/[xy]/g, function (c) {
+        var r = (Math.random() * 16) | 0,
+            v = c === "x" ? r : (r & 0x3) | 0x8;
+        return v.toString(16);
+    });
+}
```

Comment:
per other comments, this module is unnecessary, we can just use `crypto.randomUUID()` nowadays

```suggestion
```